cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesReports.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion;
+
+import com.powsybl.commons.reporter.Report;
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.commons.reporter.TypedValue;
+import com.powsybl.iidm.network.Bus;
+import com.powsybl.iidm.network.Substation;
+
+/**
+ * @author Miora Vedelago <miora.ralambotiana at rte-france.com>
+ */
+public final class CgmesReports {
+
+    private CgmesReports() {
+    }
+
+    // INFO
+    public static void importedCgmesNetworkReport(Reporter reporter, String networkId) {
+        reporter.report(Report.builder()
+                .withKey("importedCgmesNetwork")
+                .withDefaultMessage("CGMES network ${networkId} is imported.")
+                .withValue("networkId", networkId)
+                .withSeverity(TypedValue.INFO_SEVERITY)
+                .build());
+    }
+
+    // WARN
+    public static void badTargetValueRegulatingControlReport(Reporter reporter, String eqId, double targetValue) {
```

Comment:
Maybe the method name should include the word `voltage`. My suggestion is that the name of the method should always be equal to the report key, adding the suffix `Report`.